src/errors.ts

```diff
@@ -130,7 +130,9 @@ export class AwilixResolutionError extends AwilixError {
     if (typeof name === 'symbol') {
       name = (name as any).toString()
     }
-    resolutionStack = resolutionStack.slice()
+    resolutionStack = resolutionStack
+      .slice()
```

Comment:
Since you do a `map` you can remove `slice`.

Reply:
Good catch. Total face-palm on my part. I just pushed that change.